cartridges/openshift-origin-cartridge-perl/bin/build

```diff
@@ -44,6 +45,11 @@ then
 
   for f in $(( echo "$DEPS" | xargs /usr/lib/rpm/perl.req | awk '{ print $1 }' | sed 's/^perl(\(.*\))$/\1/'; cat ${CPAN_PATH} 2>/dev/null ) | sort -u | grep . )
   do
+    # Skipping well-known perl pragmas
+    for element in ${EXCLUDED_PRAGMAS[@]}; do
+      [[ $element == $f ]] && continue
```

Comment:
I don't think "continue" here will break out of the loop you intended... this is basically a no-op.